demos/src/demo.scss

```diff
@@ -1,80 +1,241 @@
-// Styles for demo UI
+$o-colors-is-silent: false;
 
-.demo-sample {
-	margin-bottom: 10px;
-	overflow: hidden;
-	white-space: nowrap;
-	text-overflow: ellipsis;
-	height: 25px;
-}
-.demo-swatch {
-	width: 50px;
-	height: 20px;
-	border-radius: 3px;
-	font-size: 20px;
-	line-height: 20px;
-	text-align: center;
-	font-style: italic;
-	display: inline-block;
-	margin-right: 10px;
-	vertical-align: middle;
-	border-width: 2px;
-	border-style: solid;
-	color: transparent;
-	border-color: transparent;
-}
-.demo-swatch:after {
-	content: 'Abc';
-}
-.demo-name {
-	font-weight: bold;
-	white-space: nowrap;
-	overflow: hidden;
-	text-overflow: ellipsis;
+// Import colors module
+@import '../../main';
+
+@function oColorsJSONifyMap($map) {
+	$json: '{';
+	$comma: false;
+
+	@each $key, $value in $map {
+		@if $comma {
+			$json: '#{$json}, ';
+		} @else {
+			$comma: true;
+		}
+
+		$json: '#{$json}"#{$key}": "#{$value}"';
+	}
+
+	$json: '#{$json}}';
+
+	@return $json;
 }
-.demo-descrip {
-	font-family: monospace;
-	margin-left: 10px;
-	font-size: 75%;
-	color: #777777;
+
+body {
+	content: "#{oColorsJSONifyMap($o-colors-palette)}";
+	box-sizing: border-box;
+	font-family: Helvetica, Arial, sans-serif;
+
+	*,
+	*:before,
+	*:after {
+		box-sizing: inherit;
+	}
 }
 
+@debug "Palette: #{$o-colors-palette}";
+@each $name, $csscolor in $o-colors-palette {
+	@if $csscolor != 'transparent' and $csscolor != 'inherit' {
+		.o-colors-palette-#{$name} .hex,
+		.o-colors-palette-#{$name}:after {
+			@if $name == 'jade' {
+				color: oColorsGetTextColor($csscolor, 95);
+			} @else {
+				color: oColorsGetTextColor($csscolor, 90);
+			}
+		}
+	}
+}
 
-// Demos - keep in sync with sass-usage.mustache where appropriate
+.o-colors-palette-org-b2c {
+	&:after,
+	.hex {
+		color: oColorsGetTextColor(oColorsGetPaletteColor('org-b2c'), 90);
+	}
+}
 
-// Remove silent mode for demo so that standard classes can be demoed in palette and use-cases demos
-$o-colors-is-silent: false;
+.o-colors-palette-org-b2c-light {
+	&:after,
+	.hex {
+		color: oColorsGetTextColor(oColorsGetPaletteColor('org-b2c-light'), 90);
+	}
+}
 
-// You can add custom use cases before importing colors module by defining the use-cases map yourself
-$o-colors-usecases: (
-	foo: (background: 'green')
-);
+.o-colors-palette-org-b2c-dark {
+	&:after,
+	.hex {
+		color: oColorsGetTextColor(oColorsGetPaletteColor('org-b2c-dark'), 90);
+	}
+}
 
-// Import colors module
-@import '../../main';
+.container {
+	width: 100%;
+	max-width: 1100px;
+	margin: 0 auto;
+}
+
+.demo-wrapper {
+	width: 100%;
+	display: flex;
+	margin-left: -10px;
+	margin-right: -10px;
+	justify-content: flex-start;
+	flex-direction: row;
```

Comment:
Can demos use other Origami components? If so, should we use o-grid and o-buttons?